test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Customer.Index.html

```diff
@@ -27,10 +27,10 @@
         <div>
             <label class="order" for="Gender">Gender</label>
             <input type="radio" value="Male" data-val="true" data-val-required="The Gender field is required." id="Gender" name="Gender" /> Male
-            <input type="radio" value="Female" id="Gender" name="Gender" /> Female
+            <input type="radio" value="Female" checked="checked" id="Gender" name="Gender" /> Female
             <span class="field-validation-valid" data-valmsg-for="Gender" data-valmsg-replace="true"></span>
         </div>
-        <div class="order validation-summary-errors" data-valmsg-summary="true"><ul><li>The value &#x27;&#x27; is invalid.</li>
+        <div class="order validation-summary-errors" data-valmsg-summary="true"><ul><li>The field Number must be between 1 and 100.</li>
```

Comment:
These changes are due to passing input that's *as-valid* as it was before my change.